spec/foo_spec.js

```diff
@@ -1,5 +1,4 @@
-require('core-js/es5');
-var expect = require( 'chai' ).expect;
```

Comment:
We might  need to revisit this deletion as it prevents Mocha from being able to run tests from command line, e.g.:
`mocha --compilers js:babel/register spec/book_spec.js`